# Internal, probably not for upstream.
# Removes gcc-specific compiler option.

--- a/ext/intl/config.m4	Wed Apr 15 11:05:57 2015
+++ b/ext/intl/config.m4	Mon Apr 27 13:30:25 2015
@@ -85,7 +85,7 @@
     breakiterator/codepointiterator_internal.cpp \
     breakiterator/codepointiterator_methods.cpp \
     idn/idn.c \
-    $icu_spoof_src, $ext_shared,,$ICU_INCS -Wno-write-strings)
+    $icu_spoof_src, $ext_shared,,$ICU_INCS)
   PHP_ADD_BUILD_DIR($ext_builddir/collator)
   PHP_ADD_BUILD_DIR($ext_builddir/converter)
   PHP_ADD_BUILD_DIR($ext_builddir/common)
